### PR TITLE
Fix camel case warnings

### PR DESCRIPTION
**Pull request by rinon**

### CODE REVIEW

File: tools/dav1d_cli_parse.rs [hunk 1]
```diff
@@ -1,3 +1,5 @@
+#![allow(non_camel_case_types)]
```
Comment (kkysen) [inline tools/dav1d_cli_parse.rs:1]: We have this is `dav1d.rs` and `seek_stress.rs` already.

File: src/recon.rs [hunk 2]
```diff
@@ -1,4 +1,5 @@
 #![deny(unsafe_op_in_unsafe_fn)]
+#![allow(non_camel_case_types)]
```
Comment (kkysen) [inline src/recon.rs:2]: We already have this in `lib.rs`; it's not needed here.  If you're seeing `rust-analyzer` warnings, that's a `rust-analyzer` issue.  It got fixed last time we upgraded our `nughtly` but then it came back at some point, so I'm not sure what the underlying issue is.

**Review by kkysen — CHANGES_REQUESTED** [review]
I don't think this is needed; we already have them in the root modules. I think it's a rust-analyzer bug.

**Comment by rinon** [comment]
I'll check with stable, but is there any real reason not to just work around this? It's been super annoying for development.

**Comment by kkysen** [comment]
> I'll check with stable, but is there any real reason not to just work around this? It's been super annoying for development.

Can I just make them camel case?  That fixes it too and it's not very hard.

**Comment by kkysen** [comment]
It's been very annoying for me too

**Comment by rinon** [comment]
Closing in favor of #1302

**Pull request closed without merging**